src/main/java/hudson/plugins/git/UserMergeOptions.java

```diff
@@ -54,6 +56,23 @@ public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStra
         this.fastForwardMode = fastForwardMode;
     }
 
+    /**
+     * @param mergeRemote remote name used for merge
+     * @param mergeTarget remote branch to be merged into current branch
+     * @param mergeStrategy merge strategy
+     * @param fastForwardMode fast forward mode
+     * @param timeout timeout before merge aborts
+     *
+     */
+    public UserMergeOptions(String mergeRemote, String mergeTarget, String mergeStrategy,
```

Comment:
I don't understand the purpose of the additional constructor.  Is there a compelling reason why the DataBoundSetter is not enough to add the timeout to a UserMergeOptions?

Reply:
Yeah I suppose the setter is more than enough to add the timeout, I just thought that if needed by a class to add the timeout via the constructor while constructing an object in the future would be helped by this constructor.
 :+1: to removing this constructor if not needed

Reply:
I think that is enough justification to keep the new constructor.  It simplifies things.